.goreleaser.yml

```diff
@@ -6,18 +6,18 @@ before:
     # you may remove this if you don't need go generate
     # - go generate ./...
 builds:
-  - id: space-poc
+  - id: space
     env:
       - CGO_ENABLED=0
-    main: ./cmd/space-poc/main.go
-    binary: space-poc
+    main: ./daemon/cmd/space/main.go
+    binary: space
     goos:
       - darwin
       - linux
   - id: installer
     env:
       - CGO_ENABLED=0
-    main: ./cmd/installer/main.go
+    main: ./daemon/cmd/installer/main.go
```

Comment:
i am wondering about adding daemon, does this mean the intention is for this to become a monorepo?

Reply:
I'm adding another folder for server-side stuff (address book, vault, etc). Maybe later we can add the FE and bundle everything in one github actions flow